contrib/exec-healthz/exechealthz.go

```diff
@@ -86,7 +87,9 @@ func (h *execWorker) start() {
 		select {
 		// If the command takes > period, the command runs continuously.
 		case <-ticker.C:
-			log.Printf("Worker running %v", *cmd)
+			if !quiet {
```

Comment:
Can you please make this a positive check? eg: 

verbose = flag(verbose, false)
if verbose { log.Printf() }